### DISCUSSION TITLE
Welcome to F5-TTS Discussions!

### DISCUSSION THREAD

**Discussion by SWivid** [SWivid/F5-TTS#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Asss466** [comment]
здравствуйте

**Reply by Han-efe** [reply]
Merhabalar bende yeni katıldım gruba. Çok mutluyum ama öğrenmek istiyorum birşey bilmiyorum;( yardımcı olursanız minnettar olurum

**Comment by cherif17** [comment]
Hola, soy cherif17 y me gustaría participar en este comunidad, conocer más información sobre todo la vida 💛, cordial saludo a todos 🖐...

**Comment by Skynettheone** [comment]
Hola! I'm Skynettheone a tech enthusiast with a problem solving amd creative thinking mind 🙂‍↔️.

**Comment by Kolkapetkinsyn** [comment]
hi. Are you planning to add languages other than English? Interested in Russian

**Reply by mamad6345** [reply]
no
![Uploading IMG_1710593759632_۲۰۲۴۰۳۱۶_۱۶۲۵۵۹_720_1280_likee.jpg…]()
